src/main/java/cc/polyfrost/oneconfig/utils/hypixel/LocrawInfo.java

```diff
@@ -111,6 +114,15 @@ public int hashCode() {
         return Objects.hash(serverId, gameMode, mapName, rawGameType);
     }
 
+    /**
+     * Compares the gametype of this locraw to another locraw.
+     * @see Comparable#compareTo(Object) 
+     */
+    @Override
+    public int compareTo(@NotNull LocrawInfo o) {
```

Comment:
how can this be compared to??????

Reply:
I'll remove it tbh it's not very helpful